stack/dev.yaml

```diff
@@ -1,4 +1,5 @@
 cpginfradev:
+  autoclass: true
```

Comment:
Accidentally included in this PR?

Reply:
Addressing both issues affecting dev stack ;)